docker/dev.dockerfile

```diff
@@ -1,27 +1,21 @@
 ARG VERSION="0.0.1-SNAPSHOT"
 
-FROM maven:3-openjdk-11-slim as dependencies
-
-RUN useradd -r -u 1001 -g root cat-user
-USER cat-user
+FROM maven:3-openjdk-11-slim as builder
 
 WORKDIR /usr/share/app
 COPY pom.xml .
 RUN mvn clean package
-
-FROM dependencies as builder
-
-WORKDIR /usr/share/app
-COPY pom.xml .
 COPY src src
 RUN mvn clean package -Dmaven.test.skip=true
 
 
 FROM openjdk:11-jre-slim-buster
 
 ARG VERSION
-ENV JAR="iudx.catalogue.server-dev-${VERSION}-fat.jar"
+ENV JAR="iudx.catalogue.server-cluster-${VERSION}-fat.jar"
```

Comment:
When maven build is done, two fatjars are created (that's how it is configured in shade pugin of pom.xml) . The dev jar is copied in dev.dockerfile and cluster jar is copied to depl.dockerfile.

Reply:
Dont copy cluster jar file to depl.dockerfile.